docs/manual/guide/production/StandAloneService.md

```diff
@@ -0,0 +1,100 @@
+# How to run a Lagom service standalone
+
+We know that Lagom doesn't prescribe any particular environment, althougth Lagom provide out of the box ConductR, an integrated environment that provide several services to deploy and manage your services, it would be interesting, in some cases, to be able to deploy a service in standalone fashion.
+
+We know that one microservice cannot live without service locator. In development environment, Lagom's plugin provided one service locator out of the box. It exclusively dedicated to the *dev mode*.  The unique thing required to run a Lagom microservice in standalone mode is a service Locator.
```

Comment:
> It exclusively dedicated to the dev mode.

Should read:

> It is exclusively for use in *dev mode*

"It" should be defined as the service locator implementation it refers to